setup.py

```diff
@@ -350,10 +351,9 @@ def get_extensions():
     has_ffmpeg = ffmpeg_exe is not None
     if has_ffmpeg:
         try:
-            ffmpeg_version = subprocess.run(
-                'ffmpeg -version | head -n1', shell=True,
-                stdout=subprocess.PIPE).stdout.decode('utf-8')
-            ffmpeg_version = ffmpeg_version.split('version')[-1].split()[0]
+            ffmpeg_version_str = str(subprocess.check_output(["ffmpeg", "-version"]))
+            ffmpeg_version = re.split(r"ffmpeg\ version\ |\.|\ |-", ffmpeg_version_str)[1:3]
```

Comment:
Thanks @vmoens !

Do we need to escape the spaces? i.e. could this just be `ffmpeg version\ |\.| |-`?

Also, let's add a small description for future-us, something like

`# this splits on both dots and spaces as the output format differs across versions / platforms`

Reply:
Space is necessary for "ffmpeg version 4.x Copyright", but @pmeier [solution](https://github.com/pytorch/vision/issues/4214#issuecomment-888168886) is actually neater on that regard IMHO

Reply:
@vmoens  do we still need to espace the spaces? Other than that, LGTM :)

Reply:
I thought i corrected that